### ISSUE TITLE
How to create filelists (.txt file)?

### ISSUE THREAD

**Issue by GauriDhande** [NVIDIA/flowtron#79]
Thank you for this amazing repository!

I would like to train the model for libritts datasets train_clean_360 and train_other_500 speakers. How to prepare filelists for these datasets?

Thanks in advance.

**Comment by cereballo** [comment]
Check out filelists/ under the root directory. The file lists for LibriTTS come with the repository.

**Comment by GauriDhande** [comment]
@cereballo the filelists of libriTTS that comes with the repository is only of train_clean_100 dataset. I would like to know how to create the same for other libritts datasets, such as train_clean_360 and train_other_500? Is there an available script in the preprocessing, or will have to write one?

**Comment by cereballo** [comment]
Ah yes that's really a matter of opening up a Jupyter notebook and doing some data wrangling :). Here's an excerpt from the Flowtron paper about how they down selected the wav files from the train-clean-100 subset:
> Speakers with less than 5 minutes of data and files that are larger than 10 seconds are filtered out. For each dataset we use at least 180 randomly chosen samples for the validation set and the remainder for the training set.

I would recommend against using the train-other-360 subset as the data is fairly dirty and you'll have a hard time training a good model with it. That being said, feel free to try! The best way to learn is by doing :).

**Comment by deepglugs** [comment]
What makes the other subset "dirty"?

**Comment by cereballo** [comment]
"Dirty" sample are generally just poor quality. This can be artifacts in the sample (e.g. pops, clicks, background noise, etc.), poor recording quality, and speech that does not match the intended transcript. You could use the dirty partition for something like speaker verification, but it's definitely too low of quality for TTS.

**Comment by rafaelvalle** [comment]
closing due to inactivity.
